src/migration/problem.orm.ts

```diff
@@ -0,0 +1,66 @@
+import { ProblemInterface } from '../api/models/problem.model';
+import { Pool } from 'mysql2';
+import pool from './pool';
+
+// Is this possible without code duplication from ProblemInterface?
+interface ProblemQueryFilter {
```

Comment:
You can take out the two properties from ProblemInterface and then extend ProblemInterface to have those two values